internal/events/aggregator.go

```diff
@@ -428,7 +428,7 @@ func (ag *aggregator) attemptMessageDispatch(ctx context.Context, msg *fftypes.M
 	valid := true
 	eventType := fftypes.EventTypeMessageConfirmed
 	switch {
-	case msg.Header.Namespace == fftypes.SystemNamespace:
+	case msg.Header.Namespace == fftypes.SystemNamespace || msg.Header.Type == fftypes.MessageTypeDefinition:
```

Comment:
Does syshandler now handle anything that is _not_ a definition?

Reply:
I think it's _only ever_ handled definitions

Reply:
I wonder if it needs a different name 🙃

Reply:
And maybe the first part of the `if` clause here is irrelevant...

Reply:
Possibly. Probably worth some input from @peterbroadhurst on this one. I'll try to chat with him when he gets a chance. The rest of the code, including the unit tests seem to be written in a way that assumes there could be system broadcasts that are _not_ definitions. Maybe there is a case I'm not thinking about, possibly a future case. Or maybe the way the code was originally written is not relevant anymore.

Reply:
Sounds good. I've approved since I think these changes are good - but probably need to have a chat about the future of what else may be on the ff_system namespace, and if "syshandler" is really the right name for the thing that is handling definitions.

Reply:
@peterbroadhurst I removed the first part of this case in the switch statement, which did change some failure code paths in the tests. I'll highlight those in the places that changed. But would appreciate your eyes on this to make sure the correct thing is still happening here.